components/gfx/paint_thread.rs

```diff
@@ -16,11 +16,10 @@ use euclid::size::Size2D;
 use font_cache_thread::FontCacheThread;
 use font_context::FontContext;
 use gfx_traits::color;
-use gfx_traits::{Epoch, LayerId, LayerKind, LayerProperties, PaintListener, PaintMsg as ConstellationMsg, ScrollPolicy};
+use gfx_traits::{Epoch, FrameTreeId, LayerId, LayerKind, LayerProperties, PaintListener, PaintMsg as ConstellationMsg, ScrollPolicy};
```

Comment:
It's strange, when I want to divide line:
'''
use gfx_traits::{color, Epoch, FrameTreeId};
use gfx_traits::{LayerId, LayerKind, LayerProperties, PaintListener, PaintMsg as ConstellationMsg, ScrollPolicy};
'''
I have an error:
''''
./components/gfx/paint_thread.rs:19: use statement is not in alphabetical order
        expected: gfx_traits::{color, Epoch, FrameTreeId}
        found: gfx_traits::{LayerId, LayerKind, LayerProperties, PaintListener, PaintMsg as ConstellationMsg, ScrollPolicy}
'''
How to divide it?